modules/porous_flow/src/userobjects/PorousFlowCapillaryPressure.C

```diff
@@ -0,0 +1,35 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowCapillaryPressure.h"
+
+template <>
+InputParameters
+validParams<PorousFlowCapillaryPressure>()
+{
+  InputParameters params = validParams<GeneralUserObject>();
+  params.addRangeCheckedParam<Real>(
+      "sat_lr",
+      0.0,
+      "sat_lr >= 0 & sat_lr <= 1",
+      "Liquid residual saturation.  Must be between 0 and 1. Default is 0");
```

Comment:
Sorry for not seeing this PR.  What is this sat_lr, by the way?  Is it the same as might enter the relative permeability, or can it be different?